cosmoz-image-viewer.html

```diff
@@ -5,86 +5,71 @@
 <link rel="import" href="../cosmoz-behaviors/cosmoz-behaviors.html">
 <link rel="import" href="../cosmoz-i18next/cosmoz-i18next.html">
 <link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html" />
+
 <script src="../photoswipe/dist/photoswipe.min.js"></script>
 <script src="../photoswipe/dist/photoswipe-ui-default.min.js"></script>
 
+<link rel="import" href="photoswipe-styles.html">
+
 <dom-module id="cosmoz-image-viewer">
 	<template>
-		<style>
```

Comment:
This template is not part of Photoswipe but how we display the image(s) inline.
Photoswipe is not activated until the "zoom" button is clicked.